src/generic/problem.h

```diff
@@ -717,6 +717,22 @@ namespace oomph
     /// with a smaller dt).
     double DTSF_min_decrease;
 
+    /// Safety factor to ensure we are aiming for a target error, TARGET,
+    /// that is below our tolerance:
+    ///     TARGET = Target_error_safety_factor * TOL
+    /// For this to make sense Target_error_safety_factor should be <1.0. If
+    /// Keep_temporal_error_below_tolerance is set to true (default) then,
+    /// without this, timesteps produced by the adaptive time-step suggester
```

Comment:
..timesteps suggested by the adaptive timestepper...

Reply:
Yep, correct. Will change.